Task1_KFox/app/scss/styles.scss

```diff
@@ -19,49 +23,255 @@ $black: #000000;
     }
 }
 
-html {
+@mixin morelink($color, $pos, $posValue, $topValue) {
+    color: $color;
+    text-decoration: none;
+    font-size: 1rem; 
+    //position: relative;
+    padding-bottom: 2px;
+    //display: inline-block;
+                   
+    &::before {
+        content: '';
+        display: block;
+        width: 20px;
+        height: 20px;
+        border: 1px $yellow solid;
+        @include border-radius(10px);
+        text-align: center;
+        @if $topValue != 0 {
+            top: $topValue;
+        }
+        @else {
+            top: 0;
+        }
+        @if $pos == left {
+            left: $posValue; 
+        } @else if $pos == right {
+            right: $posValue;
+        }
+        position: absolute;        
+    }
+
+    &::after {
+        content: '';
+        display: block;
+        width: 7px;
+        height: 7px;
+        border-top: 1px $yellow solid;
+        border-right: 1px $yellow solid;            
+        @if $topValue != 0 {
+            top: $topValue+7px;
+        }
+        @else {
+            top: 7px;
+        }
+        @if $pos == left {
+            left: $posValue+5px; 
+        } @else if $pos == right {
+            right: $posValue+9px;
+        }
+        position: absolute;
+        transform: rotate(45deg);
+    }    
+} 
+ 
+@mixin border-radius($value) {
+    -webkit-border-radius: $value;
```

Comment:
border-radius is already well supported. 

Anyway, use post processors to achieve this.
For instance https://autoprefixer.github.io/